app/models/user.rb

```diff
@@ -81,4 +89,15 @@ def active_for_authentication?
   def inactive_message
     staff_account.deactivated_at ? :deactivated : super
   end
+
+  def full_name(format = :default)
+    case format
+    when :default
+      "#{first_name} #{last_name}"
+    when :last_first
+      "#{last_name}, #{first_name}"
+    else
+      raise ArgumentError, "Unsupported format: #{format}"
+    end
+  end
```

Comment:
I was a little torn if this belonged here or in a helper. I think it makes sense in the model.